models/authentication/agent/organisation/might-be-an-administration.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * The orga belonging to following codes might be authorize to access espace agent but are not by default
+ */
+const codeJuridiquesThatMightBeAuthorized: string[] = [
```

Comment:
So those are the ones that get `page habilitation requise`?

Reply:
`connexion/habilitation/requise`